core/domain/exp_domain.py

```diff
@@ -1396,6 +1396,37 @@ def get_translatable_text(self, language_code):
 
         return state_names_to_content_id_mapping
 
+    def get_completed_translations(self, language_code):
+        """Returns all the contents along with the corresponding
+        translations for which the process of the process of translation in
+        the given language has been completed.
+
+        Args:
+            language_code: str. The language code in which translation is
+                required.
+
+        Returns:
+            dict(str, list(str)). A dict where the key is type of data
+            contained in the list i.e. translation or content and value
+            is list of strings i.e. actual data.
+        """
+        completed_translations = []
+        completed_translations_content = []
+        for state_name, state in self.states.items():
+            translations_and_ids = state.get_translated_text_and_ids(
+                language_code)
+            for content_id, translated_text in translations_and_ids:
+                completed_translations_content.append(
+                    self.get_content_html(state_name, content_id))
+                completed_translations.append(translated_text)
+        translations_and_content = collections.namedtuple(
```

Comment:
`translations_and_content` should be capitalized and in camel case like a class name.